lib/features/mailbox_dashboard/presentation/controller/mailbox_dashboard_controller.dart

```diff
@@ -1526,6 +1527,19 @@ class MailboxDashBoardController extends ReloadableController {
     dispatchAction(OpenSpamMailboxAction(context, presentationMailbox));
   }
 
+  bool get enableSpamRepot => spamReportController.enableSpamReport;
```

Comment:
Why need this assigment?

Reply:
- I need this to check when building the UI

<img width="1241" alt="Screenshot 2023-01-17 at 00 04 02" src="https://user-images.githubusercontent.com/99852347/212732415-daa0b924-3ccc-4313-8bad-260a53740f9b.png">